### PR TITLE
DDP-3871 setup for a separate appengine UI running in broad-ddp-mbc

### PR DESCRIPTION
**Pull request by zyme**
DDP-3871  calls for a temporary "we're down" page to be deployed during the migration to pepper.  This GAE page will run only in the `broad-ddp-mbc` project, where there are no other GAE apps, so we don't have to worry about trampling `dispatch.yaml`.

The plan is to switch DNS and set this page up as `mbcproject.org` once we take down the rails site and start the migration.

This branch is [deployed here](https://broad-ddp-mbc.uc.r.appspot.com/).  Take a look!

### CODE REVIEW

File: ddp-workspace/projects/ddp-mbc/outage/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        Thank you for visiting the MBCproject.org. The site is currently undergoing maintenance and improvements, which means the site will be inaccessible during this time.
+      </p>
+    </div>
+  </div>
+
+
+  <div class="row">
+    <div class="col s10 offset-s1 m8 offset-m2 l6 offset-l3">
+      <p>
+        We apologize for the inconvenience. If you wish to sign up for the project, please revisit the MBCproject.org tomorrow. If you have any questions about the project or you would like more information, please reach out to us at info@mbcproject.org or call us at 617-800-1622.
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-mbc/outage/index.html]:
```suggestion
        We apologize for the inconvenience. If you wish to sign up for the project, please revisit the MBCproject.org tomorrow. If you have any questions about the project or you would like more information, please reach out to us at <a href="mailto:info@mbcproject.org">info@mbcproject.org</a> or call us at <a href="tel:617-800-1622">617-800-1622</a>.
```

**Comment by ssettipalli** [comment]
looks good. 
Not that important but wondering if we can make this generic to use for future study migrations rather than mbc specific !!
Basically make urls and other stuff something like "cmi-study" rather than "mbc" and for future migrations all we need to change is probably index page content.

**Review by ssettipalli — APPROVED** [review]
looks good to me

**Comment by fat23cat** [comment]
@zyme, is this PR still needed, or can we close it?

**Pull request closed without merging**